docs/changes/17480.other.rst

```diff
@@ -0,0 +1 @@
+Update bundled js library datatables to current version.
```

Comment:
Is this the correct version number? "Current" will be confusing in the future, so best to state the version here.
```suggestion
Update bundled js library datatables to 2.1.8.
```